### PR TITLE
Fix bugs of Broker load

### PR DESCRIPTION
**Pull request by morningman**
1. Use same UUID as query ID and load ID of a load execution plan.
Each load execution plan has a load ID, and as a plan, there is also a query ID.
We can use same UUID as query ID and load ID, for tracing the load process more easily.

2. Change the load ID when retrying a load execution plan.
When a load execution plan retry, the load ID should be changed, otherwise BE can not
distinguish the old and new load requests.

3. Cancel the running loading task when cancelling the broker load.
When user cancel a broker load, the running loading task should also be cancelled, or
it may occupies the worker thread for a long time.

4. Remove the unnecessary query report when doing load execution plan.
Only the last query report is needed.

5. Add a new BE config `tablet_writer_rpc_timeout_sec`.
It is used for RPC of tablet sink. The default is 600 seconds. which is long enough for flushing
about 6GB data. The long timeout config will reduce the possibility of encountering `fail to send batch` error when loading.

6. Use `streaming_load_max_mb` instead of `mini_load_max_mb` in BE config.

7. Add more logs for tracing a broker load process easily.

### CODE REVIEW

File: be/src/exec/tablet_sink.h [hunk 1]
```diff
@@ -263,6 +266,8 @@ class OlapTableSink : public DataSink {
 
     // BE id -> execution time of add batch in us
     std::unordered_map<int64_t, int64_t> _node_add_batch_time_map;
+    std::unordered_map<int64_t, int64_t> _node_add_batch_wait_time_map;
+    std::unordered_map<int64_t, int64_t> _node_add_batch_num_map;
```
Comment (imay) [inline be/src/exec/tablet_sink.h]: use a struct to enclose these

File: be/src/runtime/tablet_writer_mgr.cpp [hunk 2]
```diff
@@ -309,7 +312,9 @@ Status TabletWriterMgr::add_batch(
                 << ", err_msg=" << st.get_error_msg();
         }
         if (finished) {
+            MonotonicStopWatch timer;
```
Comment (imay) [inline be/src/runtime/tablet_writer_mgr.cpp]: you should start this

File: fe/src/main/java/org/apache/doris/load/loadv2/LoadJob.java [hunk 3]
```diff
@@ -682,7 +697,7 @@ public void afterAborted(TransactionState txnState, boolean txnOperated, String
             // record attachment in load job
             executeAfterAborted(txnState);
             // cancel load job
-            executeCancel(new FailMsg(FailMsg.CancelType.LOAD_RUN_FAIL, txnStatusChangeReason), false);
+            unprotectedExecuteCancel(new FailMsg(FailMsg.CancelType.LOAD_RUN_FAIL, txnStatusChangeReason), false);
```
Comment (imay) [inline fe/src/main/java/org/apache/doris/load/loadv2/LoadJob.java]: clear idToTask

File: fe/src/main/java/org/apache/doris/load/loadv2/LoadingTaskPlanner.java [hunk 4]
```diff
@@ -201,4 +200,17 @@ private String convertBrokerDescPartitionInfo() throws LoadException, MetaNotFou
...
+    public void updateLoadId(TUniqueId loadId) {
+        for (PlanFragment planFragment : fragments) {
+            if (!(planFragment.getSink() instanceof OlapTableSink)) {
+                continue;
+            }
+            OlapTableSink olapTableSink = (OlapTableSink) planFragment.getSink();
+            olapTableSink.updateLoadId(loadId);
+        }
+
+        LOG.info("update olap table sink's load id to {}, job: {}", DebugUtil.printId(loadId), loadJobId);
```
Comment (imay) [inline fe/src/main/java/org/apache/doris/load/loadv2/LoadingTaskPlanner.java]: Delete this debug log

**Review by imay — CHANGES_REQUESTED** [review]

File: fe/src/main/java/org/apache/doris/load/routineload/KafkaTaskInfo.java [hunk 5]
```diff
@@ -99,16 +99,11 @@ protected String getTaskDataSourceProperties() {
     }
 
     private TExecPlanFragmentParams updateTExecPlanFragmentParams(RoutineLoadJob routineLoadJob) throws UserException {
```
Comment (EmmyMiao87) [inline fe/src/main/java/org/apache/doris/load/routineload/KafkaTaskInfo.java]: The better name of method is replan.

File: fe/src/main/java/org/apache/doris/load/loadv2/LoadingTaskPlanner.java (hunk 4 above)
Comment (EmmyMiao87) [reply fe/src/main/java/org/apache/doris/load/loadv2/LoadingTaskPlanner.java]: Link loadId and jobId is necessary.

File: be/src/exec/tablet_sink.h (hunk 1 above)
Comment (morningman) [reply be/src/exec/tablet_sink.h]: OK

File: be/src/runtime/tablet_writer_mgr.cpp (hunk 2 above)
Comment (morningman) [reply be/src/runtime/tablet_writer_mgr.cpp]: OK

File: fe/src/main/java/org/apache/doris/load/loadv2/LoadingTaskPlanner.java (hunk 4 above)
Comment (morningman) [reply fe/src/main/java/org/apache/doris/load/loadv2/LoadingTaskPlanner.java]: > Delete this debug log

No, this is not a debug log. We can not find the new load id without this bug

File: fe/src/main/java/org/apache/doris/load/loadv2/BrokerLoadPendingTask.java [hunk 6]
```diff
@@ -65,24 +65,33 @@ private void getAllFileStatus()
...
-                                          .add("file_status", fstatus)
-                                          .build());
+                                .add("file_status", fstatus).build());
                     }
                 }
+                totalFileSize += groupFileSize;
+                totalFileNum += fileStatuses.size();
+                LOG.info("get {} files to in file group {}. size: {}. job: {}",
+                        fileStatuses.size(), groupNum, groupFileSize, callback.getCallbackId());
+                groupNum++;
```
[the remark this answers is not in the document]
Comment (morningman) [reply fe/src/main/java/org/apache/doris/load/loadv2/BrokerLoadPendingTask.java]: I will add a table id here

File: fe/src/main/java/org/apache/doris/load/routineload/KafkaTaskInfo.java (hunk 5 above)
Comment (morningman) [reply fe/src/main/java/org/apache/doris/load/routineload/KafkaTaskInfo.java]: change it to `rePlan()`

File: be/test/exec/tablet_sink_test.cpp [hunk 7]
```diff
@@ -935,6 +935,11 @@ TEST_F(OlapTableSinkTest, decimal) {
 }
 
 int main(int argc, char* argv[]) {
+    std::string conffile = std::string(getenv("DORIS_HOME")) + "/conf/be.conf";
+    if (!doris::config::init(conffile.c_str(), false)) {
+        fprintf(stderr, "error read config file. \n");
+        return -1;
+    }
```
Comment (imay) [inline be/test/exec/tablet_sink_test.cpp]: you can set config value in SetUp(), otherwise you can only run this test in run-ut.sh

**Review by imay — APPROVED** [review]

**Pull request merged**